ide/refactoring.api/src/org/netbeans/modules/refactoring/plugins/FileMovePlugin.java

```diff
@@ -80,7 +80,7 @@ public Problem checkParameters() {
     public void cancelRequest() {
     }
     
-    private class MoveFile extends SimpleRefactoringElementImplementation {
+    public class MoveFile extends SimpleRefactoringElementImplementation {
```

Comment:
Is this new API class supposed to be subclassable? As it seems it is...

Reply:
There should be an API change note, I assume. Btw. is everything referenced - e.g. `SimpleRefactoringElementImplementation`, `RefactoringElementsBag` already public? Btw. I don't see a reason why this class was made public - searching for `MoveFile` yields no results...